UnityProject/Assets/Scripts/UI/Admin/DIMGUI/Characters/CharacterSheetEditor.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using AdminTools;
+using ImGuiNET;
+using Newtonsoft.Json;
+using UImGui;
+using UnityEngine;
+using Logs;
+using Messages.Client.Admin;
+using Shared.Managers;
+using Systems.Character;
+using Util;
+
+namespace UI.Admin.DIMGUI.Characters
+{
+	public class CharacterSheetEditor : SingletonManager<CharacterSheetEditor>
```

Comment:
how do you show this window? I didn't see any code that enables it and you don't have an OnEnable method either

Reply:
It's from a button

Reply:
can you show?

Reply:
![image](https://github.com/user-attachments/assets/4b697c44-ae25-4b49-a6f4-76b3ee403952)
![image](https://github.com/user-attachments/assets/a9a5546c-a914-4c8f-9206-a945d4e00378)